pinecone/models.go

```diff
@@ -67,10 +81,12 @@ const (
 	CollectionStatusTerminating  CollectionStatus = "Terminating"
 )
 
+// PodSpecMetadataConfig is the metadata fields to be indexed when a Pinecone Index is created.
```

Comment:
nit: should it be "are" here?

```suggestion
// PodSpecMetadataConfig are the metadata fields to be indexed when a Pinecone Index is created.
```

Reply:
Changed to just 'represents'